src/incubator/Dialog/DialogHeader.tsx

```diff
@@ -1,30 +1,42 @@
 import {isEmpty} from 'lodash';
 import React, {useMemo} from 'react';
+import {StyleSheet} from 'react-native';
 import {asBaseComponent} from '../../commons/new';
+import {Spacings, Colors, BorderRadiuses, Dividers} from 'style';
 import View from '../../components/view';
-import DialogText from './DialogText';
-import DialogKnob from './DialogKnob';
-import DialogDivider from './DialogDivider';
+import HeaderContent from './HeaderContent';
 import {DialogHeaderProps} from './types';
 
 const DialogHeader = (props: DialogHeaderProps = {}) => {
   const {text = {}, renderContent, showKnob = true, showDivider = true, ...others} = props;
 
+  const knob = useMemo(() => {
+    if (showKnob) {
+      return <View style={styles.knob}/>;
+    }
+  }, [showKnob]);
+
   const _renderContent = useMemo(() => {
```

Comment:
I would change "_renderContent" to "headerContent" to be extra clear and follow the other component's naming

Reply:
Missed that, 10x
Maybe I should rename the prop as well 🤔

Reply:
But it's under `headerProps`, WDYT?

Reply:
I think the prop name is good